packages/orchestration/src/orchestration.js

```diff
@@ -190,7 +217,9 @@ const prepareOrchestration = (zone, createChainAccount) =>
           const network = getPower(this.state.powers, 'network');
           const port = await E(network)
             .bind(`/ibc-port/icacontroller-${this.state.icaControllerNonce}`)
-            .catch(e => Fail`Failed to bind port: ${e}`);
+            .catch(e => {
+              throw Fail`Failed to bind port: ${e}`;
```

Comment:
was the explicit `throw` necessary for some type inference?

Reply:
removed